c2corg_api/views/user.py

```diff
@@ -370,7 +370,7 @@ def get(self):
         return {
             'email': user.email,
             'toponame': user.name,
```

Comment:
OK, let's simplify stuff! I will update both this PR and https://github.com/c2corg/v6_ui/pull/262.